deploy/036_deploy_mode_adapter.ts

```diff
@@ -0,0 +1,33 @@
+import { ZERO_ADDRESS } from "@uma/common";
+import { L1_ADDRESS_MAP } from "./consts";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId, network } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  const chainId = parseInt(await getChainId());
+
+  await deploy("Mode_Adapter", {
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+    args: [
+      L1_ADDRESS_MAP[chainId].weth,
+      L1_ADDRESS_MAP[chainId].modeCrossDomainMessenger,
+      L1_ADDRESS_MAP[chainId].modeStandardBridge,
+      L1_ADDRESS_MAP[chainId].usdc,
+      // L1_ADDRESS_MAP[chainId].cctpTokenMessenger,
+      // For now, we are not using the CCTP bridge and can disable by setting
+      // the cctpTokenMessenger to the zero address.
+      ZERO_ADDRESS,
+    ],
+  });
+};
+
+module.exports = func;
+func.dependencies = ["HubPool"];
```

Comment:
can we remove this `dependency`?

Reply:
We can. It is done here: https://github.com/across-protocol/contracts-v3/pull/494/commits/ec8fe263ed75eb549c709718c27619144dd3f610